pkg/controller/migmigration/task.go

```diff
@@ -212,7 +214,7 @@ var RollbackItinerary = Itinerary{
 		{Name: DeleteBackups, Step: StepFinal},
 		{Name: DeleteRestores, Step: StepFinal},
 		{Name: DeleteRegistries, Step: StepFinal},
-		{Name: EnsureStagePodsDeleted, Step: StepFinal, all: HasStagePods},
```

Comment:
I want to call out that I removed this flag so that rollback will always remove stage pods